src/ui/modal/default-modal/DefaultModal.stories.tsx

```
@@ -0,0 +1,123 @@
+function StatefulDefaultModal(props: StatefulDefaultmModalProps) {
+  const [open, setOpen] = useState(false);
+  const handleOpenChange = (open: boolean) => setOpen(open);
+  const handleClose = () => setOpen(false);
+  const handleConfirm = () => {
+    alert('handleRightClick 실행 완료');
+    setOpen(false);
+  }
+
+  return (
+    <>
+      <button onClick={() => setOpen(true)}>모달 열기</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)